### PR TITLE
Add soxx v2.0.3 with npm auto-update config

### LINKED ISSUE #7762: [Request] [Author] Add soxx
**Library name:** soxx
**Git repository url:** https://github.com/therebelrobot/soxx
**npm package url(optional):** https://npmjs.org/package/soxx
**License(s):** ISC
**Official homepage:** https://github.com/therebelrobot/soxx
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by x09326**
PR for #7762 
@pvnr0082t please help me review it, thanks.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/therebelrobot/soxx
 * Official website (optional, not the repository):
 * NPM package url (optional): https://npmjs.org/package/soxx
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 1
   - Count of watchers: 1
   - Count of forks: 0
 * NPM download stats (optional):
   - Downloads in the last day: 30
   - Downloads in the last week: 260
   - Downloads in the last month: 310

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #7762, cc @therebelrobot

### CODE REVIEW

File: ajax/libs/soxx/2.0.3/soxx.min.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
...
+//# sourceMappingURL=./2.0.3/soxx.min.js.map
```
Comment (pvnr0082t) [inline ajax/libs/soxx/2.0.3/soxx.min.js]: @x09326 I think you don't update your minify tool. This is Wrong.

**Comment by x09326** [comment]
@pvnr0082t 
I have updated it, please help me review it again, thanks.

**Comment by pvnr0082t** [comment]
@PeterDaveHello I think this PR is ok. Please review it again, thank you.

File: ajax/libs/soxx/2.0.3/soxx.min.js (hunk 1 above)
Comment (PeterDaveHello) [inline ajax/libs/soxx/2.0.3/soxx.min.js]: Nice catch!

**Comment by PeterDaveHello** [comment]
Thanks for your contribution.

**Pull request merged**